pkgs/applications/blockchains/zcash/default.nix

```diff
@@ -49,15 +113,19 @@ rustPlatform.buildRustPackage.override { inherit stdenv; } rec {
     configureFlagsArray+=("RUST_VENDORED_SOURCES=$cargoDepsCopy")
   '';
 
-  CXXFLAGS = [
-    "-I${lib.getDev utf8cpp}/include/utf8cpp"
-    "-I${lib.getDev cxx-rs}/include"
-  ];
+  preConfigure = ''
+    export CFLAGS="-pipe -O3 -Wno-unknown-warning-option"
+    export CXXFLAGS="-pipe -O3 -Wno-unknown-warning-option -I${lib.getDev utf8cpp}/include/utf8cpp -I${lib.getDev cxx-rs}/include"
+  '';
+
+  NIX_HARDENING_ENABLE = "";
```

Comment:
Is there no boolean flag for this?

Reply:
None that I could find. The configure script adds the proper flags automatically:
```
zcash>   CC            = clang
zcash>   CFLAGS        = -g -O2
zcash>   CPPFLAGS      =   -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2  -DHAVE_BUILD_INFO -D__STDC_FORMAT_MACROS
zcash>   CXX           = clang++ -std=c++17
zcash>   CXXFLAGS      =  -g  -Wformat -Wformat-security -Wstack-protector -fstack-protector-all -Wthread-safety-analysis  -Werror  -g -O2 -fwrapv -fno-strict-aliasing
```

Reply:
```suggestion
  hardeningEnable = [ ];
```